android-project/app/src/main/java/oceantreasur/es/fragments/AnimationController.java

```diff
@@ -0,0 +1,135 @@
+package oceantreasur.es.fragments;
+
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.animation.Animation;
+import android.view.animation.TranslateAnimation;
+import android.widget.RelativeLayout;
+
+import java.util.ArrayList;
+import java.util.Random;
+
+import oceantreasur.es.R;
+import oceantreasur.es.network.OceanTreasuresApplication;
+
+import static oceantreasur.es.R.dimen.small_fish_height;
+import static oceantreasur.es.R.dimen.small_fish_width;
+import static oceantreasur.es.R.layout.big_fish;
+import static oceantreasur.es.R.layout.small_fish;
+import static oceantreasur.es.view.AnimationConstants.DURATION_OF_BIG_FISH_ANIMATION;
+import static oceantreasur.es.view.AnimationConstants.FISH_MARGIN_BOTTOM;
+import static oceantreasur.es.view.AnimationConstants.FISH_MARGIN_TOP;
+import static oceantreasur.es.view.AnimationConstants.MAX_BIG_FISHES;
+import static oceantreasur.es.view.AnimationConstants.MAX_DURATION;
+import static oceantreasur.es.view.AnimationConstants.MAX_SMALL_FISH;
+import static oceantreasur.es.view.AnimationConstants.MAX_SPAWNING_POINT_IN_PERCENTS;
+import static oceantreasur.es.view.AnimationConstants.MAX_TIME_OFFSET_BIG_FISH;
+import static oceantreasur.es.view.AnimationConstants.MAX_TIME_OFFSET_SMALL_FISH;
+import static oceantreasur.es.view.AnimationConstants.MIN_DURATION;
+import static oceantreasur.es.view.AnimationConstants.MIN_SPAWNING_POINT_IN_PERCENTS;
+import static oceantreasur.es.view.AnimationConstants.MIN_TIME_OFFSET_BIG_FISH;
+import static oceantreasur.es.view.AnimationConstants.MIN_TIME_OFFSET_SMALL_FISH;
+import static oceantreasur.es.view.AnimationConstants.SPAWNING_POINT_OF_BIG_FISH;
+import static oceantreasur.es.view.ScreenUtils.getDimensionOfFishInPx;
+import static oceantreasur.es.view.ScreenUtils.getScreenHeight;
+import static oceantreasur.es.view.ScreenUtils.getScreenWidth;
+
+
+public class AnimationController {
+
+    private ArrayList<View> smallFishViews = new ArrayList<>();
+    private ArrayList<View> bigFishViews = new ArrayList<>();
+    private int smallFishBeginOffset = getDimensionOfFishInPx(small_fish_width);
+    private int bigFishBeginOffset = getDimensionOfFishInPx(R.dimen.big_fish_width);
+
+    AnimationController(RelativeLayout background) {
```

Comment:
Pass background and foreground ViewGroups. Big fish should always be in the background.